jubatus/server/server/classifier.idl

```diff
@@ -59,7 +59,7 @@ service classifier {
   #-
   #-  Get all labels in the model
   #@random #@nolock #@pass
-  list<string> get_labels()
+  map<string, int> get_labels()
```

Comment:
Oh.., I didn’t know that I can use ``ulong`` in IDL.
I will create new PR.
Because if regenerate codes again, commit logs will be dirty.